kernel/dev/rtc/rtc.cpp

```diff
@@ -136,23 +139,31 @@ void read_rtc() {
     }
 
     // Calculate the full (4-digit) year
-    if(RTC_CURRENT_CENTURY != 0) {
+    if (RTC_CURRENT_CENTURY != 0) {
         rtc_year += rtc_century * 100;
     } else {
         rtc_year += (RTC_CURRENT_YEAR / 100) * 100;
-        if(rtc_year < RTC_CURRENT_YEAR) rtc_year += 100;
+        if (rtc_year < RTC_CURRENT_YEAR)
+            rtc_year += 100;
     }
 }
+static uint64_t getJulianDay(uint8_t days, uint8_t months, uint16_t years)
+{
+    return (1461 * (years + 4800 + (months - 14) / 12)) / 4 + (367 * (months - 2 - 12 * ((months - 14) / 12))) / 12 - (3 * ((years + 4900 + (months - 14) / 12) / 100)) / 4
```

Comment:
If possible, I would like to see this one line split up into multiple variables. It will get optimized by the compiler anyways, so I would rather have readability.